src/computedFn.ts

```diff
@@ -71,9 +75,10 @@ export function computedFn<T extends (...args: any[]) => any>(
             return fn.apply(this, args)
         }
         // create new entry
+        let latestValue: ReturnType<T> | undefined
         const c = computed(
             () => {
-                return fn.apply(this, args)
+                return (latestValue = fn.apply(this, args))
```

Comment:
would you mind only setting `latestValue` if `opts.onCleanup` is specified, to not unnecessarily hold on to old values in memory?

Reply:
It's following how `createTransformer` works.

The `computed` would have been holding the same value already if the returned value doesn't equal to the previous value. On the other hand, if the returned value is a new object but considered equal, this will indeed hold the new value unnecessarily while the `computed` itself holds the old one, but in that case, if we still update `latestValue`, then `onCleanup` is going to get a different object than what the callers of `computedFn` get, which doesn't feel quite right either.

The `onCleanup` in `createTransformer` (and this impl for `computedFn`) feels a bit of a leaky abstraction that it's only called when each input is no longer observed. If the transformer (or computed function) is triggered by any observables internally, `onCleanup` would not be ran for the result there. So if they are observing anything internally, there will be a problem. I'm not sure what's the best solution for that.

Reply:
Ah yes good points on the computed internal state. Ok, lgtm!